src/Pages/ProductList/ProductList.js

```diff
@@ -1,31 +1,57 @@
-import React, { useState } from "react";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  filterToggle,
+  getRoomsInfo,
+} from "../../modules/ProductList/productList";
 import styled from "styled-components";
 import ProductListMain from "./ProductListMain/ProductListMain";
+import AsideMap from "./AsideMap/AsideMap";
+import { productListAPI } from "../../config";
 
-function ProductList() {
-  const [activeFilterIndex, setActiveFilterIndex] = useState(null);
+export default function ProductList() {
+  const dispatch = useDispatch();
+  const { checkedOptions, currentPage } = useSelector(
+    ({ productList: { checkedOptions, currentPage } }) => ({
+      checkedOptions: checkedOptions,
+      currentPage: currentPage,
+    })
+  );
```

Comment:
```suggestion
  const { checkedOptions, currentPage } = useSelector(
    ({ productList: { checkedOptions, currentPage } }) => ({ checkedOptions, currentPage })
  );
```
객체 단축 속성명 쓰면 이렇게도 가능한데 저는 함수 바디보다 머리가 커지는거 같아서
```suggestion
  const { checkedOptions, currentPage } = useSelector(
    ({ productList }) => ({
      checkedOptions: productList.checkedOptions,
      currentPage: productList.currentPage,
    })
  );
```
이 방식을 더 선호하긴 합니다 테이스트에 맞춰 써주세요